services/digest_io.py

```
@@ -6,246 +6,141 @@
+            digest = self._enhance_with_ai(target_date, digest, ai)
             final_path = self.blogs_dir / target_date / f"FINAL-{target_date}_digest.json"
-            from services.utils import CacheManager
-            cache_manager = CacheManager()
-            cache_manager.atomic_write_json(final_path, enhanced_digest, overwrite=True)
-            
+            self.cache.atomic_write_json(final_path, digest, overwrite=True)
             logger.info(f"Created FINAL digest: {final_path}")
-            return enhanced_digest
-            
+            return digest
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)